bin/rbenv-uninstall

```diff
@@ -81,6 +81,7 @@ for hook in "${before_hooks[@]}"; do eval "$hook"; done
 if [ -d "$PREFIX" ]; then
   rm -rf "$PREFIX"
   rbenv-rehash
+  echo "Successfully uninstalled $PREFIX"
```

Comment:
Can you remove this? A Unix command usually shouldn't print that it did something successfully, unless it failed.